src/main/java/com/epam/training/auction_backend/controllers/AuctionService.java

```diff
@@ -0,0 +1,61 @@
+package com.epam.training.auction_backend.controllers;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.apache.log4j.Logger;
+import org.hibernate.Query;
+import org.hibernate.Session;
+
+import com.epam.training.auction_backend.entity.Auction;
+
+public class AuctionService {
+	
+	public static final Logger logger = Logger.getLogger(AuctionService.class);
+	
+	public List<Auction> getActiveAuctions() {
+		return getAuctions(true);
+	}
+
+	public List<Auction> getArchivedAuctions() {
+		return getAuctions(false);
+	}
+
+	public Optional<Auction> getAuctionById(long id) {
+		Optional<Auction> auction = Optional.empty();
+		try(Session session = SessionProvider.getSession()) {
```

Comment:
I think it is better instead of using the session directly you could create an interface that etends CrudRepository and inject it in the sprint application. Like this:

@Transactional
public interface AuctionRepository extends CrudRepository<Auction, Long>{
	public List<Auction> findAll();
	public Auction findById(long id);
}